### ISSUE TITLE
Python 3.11 wheels

### ISSUE THREAD

**Issue by tomwhite** [fastlmm/bed-reader#12]
It would be very useful to have wheels for Python 3.11

**Comment by CarlKCarlK** [comment]
To use sklearn as an example, I see that it has dropped support for Python 3.7 and supports 3.8 .to 3.11.

Are you OK with bed-reader dropping 3.7 soon?

From: Tom White ***@***.***>
Sent: Wednesday, May 3, 2023 4:14 AM
To: fastlmm/bed-reader ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [fastlmm/bed-reader] Python 3.11 wheels (Issue #12)
Importance: High


It would be very useful to have wheels for Python 3.11

-

**Comment by tomwhite** [comment]
> Are you OK with bed-reader dropping 3.7 soon?

Yes, we've dropped support for 3.7 in sgkit already. Thanks @CarlKCarlK.

**Comment by CarlKCarlK** [comment]
OK, version 3.11 is now supported with v 0.2.35. Thanks for the request.

**Comment by tomwhite** [comment]
Great - thanks for the quick turnaround!
